### ISSUE TITLE
Should be able to run gauge taiko tests

### ISSUE THREAD

**Issue by NivedhaSenthil** [getgauge/taiko#430]
### Expected behavior
Run gauge taiko tests seamlessly 

### Actual behavior
Throws error 
```
error: unknown option `--run
Usage: gauge [options]
       taiko <file> [options]

Options:
  -v, --version                 output the version number
  -o, --observe                 enables headful mode and runs script with 3000ms delay by default.
          			pass --wait-time option to override the default 3000ms

  -w, --wait-time <time in ms>  runs script with provided delay

  --emulate-device <device>     Allows to simulate device viewport. Visit https://github.com/getgauge/taiko/blob/master/lib/device.js for all the available devices

  -h, --help                    output usage information
Error ----------------------------------

Connection to runner with Pid 5126 lost. The runner probably quit unexpectedly. Inspect logs for potential reasons. Error : read tcp 127.0.0.1:51866->127.0.0.1:51868: use of closed network connection
```

### Steps to reproduce
1. `gauge init js`
2. install taiko from master
3. `gauge run`

### Versions
taiko commit `b26eb83`

**Comment by NivedhaSenthil** [comment]
Fix done via 9cae987

**Comment by Debashis9012** [comment]
Tested this issue with the above steps to reproduce. We are able to run gauge taiko tests without any errors. Its has been tested with the below version.
``` 9cae987 ```
